modules/doc/content/getting_started/installation/manual_installation_llvm.md

```diff
@@ -34,7 +34,25 @@ export LD_LIBRARY_PATH=$PACKAGES_DIR/llvm-__LLVM__/lib:$LD_LIBRARY_PATH
 
 !include manual_mpich_llvm.md
 
-!include getting_started/installation/manual_miniconda.md
+## Python (possibly optional)
+
+Building MOOSE requires having Python headers (python-devel) and the Python package: `packaging`
+available. If you know you have this requirement satisfied, and you do not plan on also using
+Peacock (next section) you can skip this section.
+
+These requirements are easily installed via Conda.
+
+!include getting_started/installation/install_miniconda.md
+
+Next, install the necessary `packaging` module:
+
+```bash
+mamba install packaging
+```
+
+## Peacock (optional)
+
+!include installation/install_peacock.md
```

Comment:
Since this is used twice - is it better to put this content in a single file we include? Even if it results in some patching pain.